openwisp_radius/api/views.py

```diff
@@ -71,6 +77,27 @@ class TokenAuthentication(BaseAuthentication):
     def authenticate(self, request):
         self.check_organization(request)
         uuid, token = self.get_uuid_token(request)
+        if not uuid and not token:
+            try:
+                radtoken = RadiusToken.objects.get(
+                    user__username=request.data.get('username')
+                )
+            except RadiusToken.DoesNotExist:
+                message = (
+                    'User radius token does not exist & organization '
+                    'uuid & token not provided in request'
+                )
+                logger.warning(message)
+                # TODO:N Returning 403 not 401 as suggested in the issue
+                # rest_auth: WWW-Authenticate header for 401 responses, else coerce to 403
+                # After some reading on it, I agree, details missing is 403.
```

Comment:
In the issue it's mention this case should return `401` but rest_auth suggests, or rather forces to return `403` because auth failed due to missing information.